src/bin/sol-fbp-generator/main.c

```diff
@@ -1948,12 +1949,6 @@ parse_args(int argc, char *argv[])
         return false;
     }
 
-    if (!has_json_file) {
-        fprintf(stderr, "At least one JSON file containing the declaration of the nodes"
```

Comment:
Would be nice to still have a check for whether any JSON file was found (after taking into account all the paths), after parsing arguments and handling the environment variable.